sale_order_line_description/tests/test_sale.py

```diff
@@ -57,6 +58,7 @@ def test_check_sale_order_line_description(self):
         sale_order_line = self.sale_order_line_model.with_user(self.user_1).create(
             line_values.copy()
         )
+        sale_order_line._onchange_product_id_warning()
```

Comment:
After having changed to _compute_name() override, you don't need this anymore

Reply:
Thank you for review this PR and teach me my mistakes, I appreciate it very much because I am learning.
Pls check my changes and let me know if any further changes are needed.
Greetings!